pkg/runtime/wfengine/activity.go

```diff
@@ -52,7 +52,6 @@ type ActivityRequest struct {
 
 type activityState struct {
 	EventPayload []byte
-	Generation   uint64
```

Comment:
There's also a Generation field on the `ActivityRequest` struct. Can that be removed too?